modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -49,7 +49,7 @@ export default function UpperToolbar({
 	};
 
 	const onInputBlur = () => {
-		if (selectedLanguageId) {
+		if (selectedLanguageId && definitionTitle) {
```

Comment:
@ramonmsp Can we sort this statement `(definitionTitle && selectedLanguageId)`? Thx